src/ga.css

```diff
@@ -5,5 +5,6 @@
 @import 'styles/dropdown.css';
 @import 'styles/form.css';
 @import 'styles/header.css';
+@import 'styles/form.css';
```

Comment:
We already have `form.css` but we'll need `input.css` here.

Reply:
I might have lost it in the rebase. sorry! Just updating it!